projects/bad_example_mixed_sanitizers/build.sh

```diff
@@ -0,0 +1,92 @@
+#!/bin/bash -eu
+
+# Enable multiple sanitizers because I can.
+export CFLAGS="-O1 -fsanitize=address,undefined -fsanitize-coverage=trace-pc-guard,trace-cmp"
+export CXXFLAGS="-O1 -fsanitize=address,undefined -fsanitize-coverage=trace-pc-guard,trace-cmp -stdlib=libc++"
+
+./configure
+make -j$(nproc) clean
+make -j$(nproc) all
+
+$CXX $CXXFLAGS -std=c++11 -I. \
+    $SRC/zlib_uncompress_fuzzer.cc -o $OUT/zlib_uncompress_fuzzer \
+    -lFuzzingEngine ./libz.a
+
+#
+# How to detect.
+# 1) Get the following values:
+#    ASAN_CALLS = `objdump -dC FUZZER | egrep "callq\s+[0-9a-f]+\s+<__asan" -c`
+#    MSAN_CALLS = `objdump -dC FUZZER | egrep "callq\s+[0-9a-f]+\s+<__msan" -c`
+#    UBSAN_CALLS = `objdump -dC FUZZER | egrep "callq\s+[0-9a-f]+\s+<__ubsan" -c`
+#
+# 2) If $SANITIZER == address, the following condition should be TRUE:
+#    ASAN_CALLS > 1000 && MSAN_CALLS < 100 && UBSAN_CALLS < 250
```

Comment:
I'm afraid that there can be some non reachable code or a name collision in the binary, so not sure if  0 is a safe choice.  However, I don't mind to start it with 0, and then increase if we get any problems.

Regarding UBSan calls, I think that happens due to possibility to use UBSan together with ASan or with MSan. ASan / MSan functions call some UBSan functions, so those get compiled and linked in.